examples/hitl/rearrange_v2/world.py

```diff
@@ -165,3 +288,47 @@ def is_any_agent_holding_object(self, object_id: int) -> bool:
                 return True
 
         return object_id in self._all_held_object_ids
+
+    def _try_initialize_object_states(
+        self, sim: RearrangeSim, config: DictConfig
+    ) -> None:
+        """
+        Try initializing object states and categories. Depends on the 'habitat_llm' external package.
+        """
+        try:
+            # Initialize object state machine.
+            from habitat_llm.sims.collaboration_sim import (
+                initialize_object_state_machine,
+            )
+            from habitat_llm.sims.metadata_interface import (
+                MetadataInterface,
+                get_metadata_dict_from_config,
+            )
+
+            metadata_dict = get_metadata_dict_from_config(
+                dataset_config=config.habitat.dataset
+            )
+            metadata_interface = MetadataInterface(metadata_dict)
+            metadata_interface.refresh_scene_caches(
+                sim, filter_receptacles=False
```

Comment:
I think we are filtering everywhere else for the world_graph, so agent's won't try to set states on filtered Furniture. :thinking: 
Let's double check that or HitL folks could set states that agents wouldn't.

Reply:
Agreed this should probably be true. In hab-llm `main` `CollaborationSim.reconfigure()` sets `filter_receptacles=True` before the start of every episode.

Let's keep this consistent.